website/content/api-docs/variables/locks.mdx

```diff
@@ -0,0 +1,210 @@
+release calls, the variable will remain unlockable for at least the lock delay. 
+
+#### Sample Request
+
+```shell-session
+$ curl \
+    -XPUT -d@spec.nsv.json \
+    https://localhost:4646/v1/var/example/first?lock-renew
+```
+
+#### Sample Payload
+
+```json
+{
+  "Path": "example/first",
+  "Namespace": "prod",
+  "Lock": {
+    "ID": "670c7248-e2ef-f982-e4c5-f4437f75f1e4"
+  }
+}
+```
+
+#### Sample Response
+
+The response body only returns metadata created by the server and the lock
+parameters:
+
+```json
+{
+  "CreateIndex": 11,
+  "CreateTime": 1694555280887153000,
+  "Lock": {
+    "TTL": "15s",
+    "LockDelay": "15s",
+    "ID": "670c7248-e2ef-f982-e4c5-f4437f75f1e4"
+  },
+  "ModifyIndex": 43,
+  "ModifyTime": 1694556175092779000,
+  "Namespace": "prod",
+  "Path": "example/first"
+}
+```
+
+- `lock-release`: A call to the endpoint with the `lock-release` operation will
+immediately remove the lock over the variable, making it modifiable without 
+restrictions again. 
+
+The lock-release operation will not override the variable items, if the request
+body contains any item, it will generate a bad request response.
+
+#### Sample Request
+
+```shell-session
+$ curl \
+    -XPUT -d@spec.nsv.json \
+    https://localhost:4646/v1/var/example/first?lock-release
+```
+
+#### Sample Payload
+
+```json
+{
+  "Path": "example/first",
+  "Namespace": "prod",
+  "Lock": {
+    "ID": "670c7248-e2ef-f982-e4c5-f4437f75f1e4"
+  }
+}
+```
+
+#### Sample Response
+
+The response body returns the released variable along with metadata
+created by the server:
+
+```json
+{
+  "CreateIndex": 11,
+  "CreateTime": 1694555280887153000,
+  "ModifyIndex": 66,
+  "ModifyTime": 1694556922600469000,
+  "Namespace": "prod",
+  "Path": "example/first"
+}
+```
+
+### Sample Response for Conflict
+
+In the case of an attempt to lock, renew or modify a locked variable
+without the correct ID, the API will return HTTP error code
+409 and a response body showing the conflicting variable. If the provided ACL
+token does not also have `read` permissions to the variable path, the response
+will include only metadata and not the `Items` field:
+
+```json
+{
+  "CreateIndex": 0,
+  "CreateTime": 0,
+  "Items": null,
+  "Lock": null,
+  "ModifyIndex": 0,
+  "ModifyTime": 0,
+  "Namespace": "default",
+  "Path": "example/first"
+}
+```
+
+## Restrictions
+
+When creating a new variable using the lock-acquire operation, all the known 
+[restrictions][] regarding the path and size of the content apply, but unlike 
+regular variables, locked variables can be created with or without any items.
+
+The lock TTL and Delay must be values between 10 seconds and 24 hours.
+
+[Variables]: /nomad/docs/concepts/variables
+[`nomad var`]: /nomad/docs/commands/var
+[blocking queries]: /nomad/api-docs#blocking-queries
+[required ACLs]: /nomad/api-docs#acls
+[RFC3986]: https://www.rfc-editor.org/rfc/rfc3986#section-2
```

Comment:
Are these meant to be in markdown link format? If so, I think the

```
[Variables](/nomad/docs/concepts/variables)
[`nomad var`](/nomad/docs/commands/var)
```

convention is only one I've used (but I am not a markdown expert!)